mavros/08cd181/08cd181.bug

```diff
@@ -25,13 +25,13 @@ bug:
   detected-by: developer
   reported-by: contributor
   issue: https://github.com/mavlink/mavros/issues/161
-  time-reported: 2014-11-02 (00:00)
-  reproducibility: N/A
+  time-reported: 2014-11-02T00:00:00+00:00
+  reproducibility: always
   trace: N/A
 fix:
   repo: https://github.com/mavlink/mavros
   hash: 08cd18164e729b0fb0ed1f6b553e5458eb9b6a4c
-  pull-request: N/A
+  pull-request:
```

Comment:
Can we change the value of this field to `null`?

Reply:
I've indeed been setting all fields we don't know the value of / can't set to a meaningful value to `null`.

Reply:
So you were not following the taxonomy-description @gavanderhoorn  :).  I have no problem with null in principle, but in all the files that I have created, and presumably others who followed the description, null was not used.  This is not about this field only -- there are others that admit an empty value. 

We can agree to go with nulls, but then, I think, we should do it consistently with all fields that admit empty, update the schema, and do the massive compliance update across the repo...